src/paymentservice/charge.js

```diff
@@ -12,12 +12,14 @@ const logger = require('./logger');
 const tracer = trace.getTracer('paymentservice');
 const meter = metrics.getMeter('paymentservice');
 const transactionsCounter = meter.createCounter('app.payment.transactions')
+const Bugsnag = require("@bugsnag/js");
 
 module.exports.charge = async request => {
   const span = tracer.startSpan('charge');
 
   await OpenFeature.setProviderAndWait(flagProvider);
   if (await OpenFeature.getClient().getBooleanValue("paymentServiceFailure", false)) {
+    Bugsnag.notify(new Error("PaymentService Fail Feature Flag Enabled"));
```

Comment:
Bugsnag by default should report unhandled errors. If you remove this line, does it still send an error?

Reply:
I couldn't manage to make it work without it. But having a look into the documentation, I understood that only unhandled exceptions and unhandled promises would be reported, so I assumed that it's the expected behaviour.
![image](https://github.com/user-attachments/assets/cfa0d0fd-2e4b-497c-87c8-a51d43d16d21)